bundle/Form/FieldTypeHandler/User.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+
+use Ibexa\Contracts\Core\FieldType\Value;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinition;
+use Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+use Netgen\Bundle\InformationCollectionBundle\Form\Type\FieldType\UserCreateType;
+use Netgen\Bundle\InformationCollectionBundle\Form\Type\FieldType\UserUpdateType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class User extends FieldTypeHandler
```

Comment:
Remove this, we won't need it for info collection.

Reply:
Removed in 932f3035